lib/smart_proxy_openscap/openscap_lib.rb

```diff
@@ -13,10 +13,21 @@
 require 'json'
 require 'proxy/error'
 require 'proxy/request'
+require 'smart_proxy_openscap/openscap_exception'
 
 module Proxy::OpenSCAP
   extend ::Proxy::Log
 
+  def self.get_policy_content(policy_id)
+    policy_store_dir = File.join(Proxy::OpenSCAP::Plugin.settings.contentdir, policy_id.to_s)
+    policy_scap_file = File.join(policy_store_dir, "#{policy_id}_scap_content.xml")
+    FileUtils.mkdir_p(policy_store_dir) # will fail silently if exists
```

Comment:
so I suppose it's still useful if the directory is changed in config file and foreman-proxy has permissions on that path, if it's the case it would be nice to catch permission exception and let the user know what was the directory that it tried to create, I had to get to code to understand what was failing